src/main/java/org/assertj/core/internal/Strings.java

```diff
@@ -527,6 +529,51 @@ public void assertContainsIgnoringCase(AssertionInfo info, CharSequence actual,
       throw failures.failure(info, shouldContainIgnoringCase(actual, sequence));
   }
 
+  /**
+   * Verifies the given {@code CharSequence} has the strings, ignoring newlines.
+   *
+   * @param info contains information about the assertion.
+   * @param actual the actual {@code CharSequence}.
+   * @param values the values to look for.
+   * @throws NullPointerException if the given sequence is {@code null}.
+   * @throws IllegalArgumentException if the given values is empty.
+   * @throws AssertionError if the given {@code CharSequence} is {@code null}.
+   * @throws AssertionError if actual {@code CharSequence} doesn't have sequence
+   */
+  public void assertContainsIgnoringNewLines(final AssertionInfo info, final CharSequence actual, final CharSequence... values) {
+    doCommonCheckForCharSequence(info, actual, values);
+    final String actualNoNewLines = removeNewLines(actual);
+    final Stream<CharSequence> notFoundStream = stream(values);
+    final Stream<CharSequence> notFoundStream2 = mapContainsIgnoringNewLines(notFoundStream);
```

Comment:
inlining both `notFoundStream` and `mapContainsIgnoringNewLines` would make the code clearer I think, here it's confusing to have `notFoundStream` and `notFoundStream2`.
might look like:
```java
Set<CharSequence> notFound = stream(values).map(Strings::removeNewLines)
                                           .filter(value -> !stringContains(actualNoNewLines, value))
                                           .collect(toCollection(LinkedHashSet::new));
```